openreview/conference/templates/decision_process.py

```diff
@@ -0,0 +1,44 @@
+def process_update(client, note, invitation, existing_note):
+
+    CONFERENCE_ID = ''
+    SHORT_PHRASE = ''
+    AUTHORS_NAME = ''
+    ACCEPTED_AUTHORS_NAME = 'Authors/Accepted'
+    EMAIL_AUTHORS = False
+    baseUrl='https://openreview.net'
+
+    forum_note = client.get_note(note.forum)
+   
+    AUTHORS_ID = CONFERENCE_ID + '/Paper' + str(forum_note.number) + '/' + AUTHORS_NAME
+    
+    # Check if the content of the existing note is the same as the updated note
+    if (existing_note):
+      equal = True
+      for key in existing_note.content:
+        if existing_note.content[key] != note.content[key]:
```

Comment:
it seems the indentation is off here, are the process function working?

Reply:
Yes, I believe it is working now even though the indentation does look shallow